IFC4x3/Constants/b/BLOSSCURVE_0hYbIcTB5DJhg8TxL3hntC.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BLOSSCURVE_0hYbIcTB5DJhg8TxL3hntC" Name="BLOSSCURVE" UniqueId="2b8a54a6-74b1-4d4e-ba88-77b543af1dcc" />
```

Comment:
Yes this is a correct observation and it was done on purpose. See commit comment "Cant segment constants disentangled". It was caused by the current Enum constants naming in RC4 for IfcAlignmentHorizontalSegmentTypeEnum and IfcAlignmentCantSegmentTypeEnum. Having only on (IfcDoc) constant for BLOSSCURVE is a little bit unfortunate for proper documentation of high performance transition bends.

In EN 13803 it is stated that for Linear (aka Clothoid) transition, Helmert, Bloss, Cosine and Sine the base formula for curvature variations and the base formula for cant transitions are structurally identical. Viennese bend is the one example where formula for horizontal and the formula for cant transitions are structurally different.

BUT: although the formulas are structurally the same, different symbols are in use. Small greek "kappa" for curvature and capital D for cant. So the formulas look different. There is a commit due adding the cant formulas this evening.

Fortunately in RC4 we already use different names for the linear transition, namely CLOTHOID in horizontal layout and LINEARTRANSITION in cant layout, so it is not an issue there.

I see two options:

1) keep two seperate, nevertheless identically named constants, one for the horizontal documentation and one for the cant documentation.

2) rename the constants for cant. E.g. BLOSSCURVECANT instead of BLOSSCURVE .

I need guidance here.

PS: please note that a similar issue exists for CLOTHOID in horizontal and vertical layouts. The documentation for CLOTHOID in vertical will be different from the documentation for CLOTHOID in horizontal.

Reply:
Ahh, sorry. Forgot about that. No need for removing then.